test_microvenv.py

```diff
@@ -79,6 +79,7 @@ def test_lib64(full_venv, micro_venv):
         assert micro_lib64.exists()
         assert micro_lib64.is_symlink()
         assert micro_lib64.resolve() == (micro_venv / "lib")
+        assert pathlib.Path.readlink(micro_lib64).is_relative_to("lib")
```

Comment:
`is_relative_to()` is a bit too flexible to make sure this doesn't change, so I would prefer to be a bit strict about exactly what the symlink is set to.

```suggestion
        assert os.fsdecode(pathlib.Path.readlink(micro_lib64)) == "lib"
```